frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelConversionWindowFilter.tsx

```diff
@@ -0,0 +1,68 @@
+import { InputNumber, Row, Select, Tooltip } from 'antd'
+import { InfoCircleOutlined } from '@ant-design/icons'
+import { pluralize } from 'lib/utils'
+import React from 'react'
+import { useActions, useValues } from 'kea'
+import { funnelLogic } from 'scenes/funnels/funnelLogic'
+
+export interface FunnelConversionWindow {
+    unit?: TimeUnit
+    days?: number | undefined
+}
+
+export enum TimeUnit {
+    Day = 'Day',
+    Week = 'Week',
+}
+
+export function FunnelConversionWindowFilter(): JSX.Element {
+    const { conversionWindow, conversionWindowValueToShow } = useValues(funnelLogic)
+    const { setConversionWindow } = useActions(funnelLogic)
+
+    const options = [
+        {
+            label: pluralize(conversionWindowValueToShow, 'day', 'days', false),
+            value: TimeUnit.Day,
+        },
+        {
+            label: pluralize(conversionWindowValueToShow, 'week', 'weeks', false),
+            value: TimeUnit.Week,
+        },
+    ]
+
+    return (
+        <Row
```

Comment:
Really briefly looked at our backend code (@neilkakkar pls do lmk if im completely wrong) but it looks like the [default window is 14 days](https://github.com/PostHog/posthog/blob/e9711b0454f8135399ffc7bdb07be55eb2121ff4/ee/clickhouse/queries/funnels/base.py#L37-L38) if none of the parameters are specified. Turning this off would technically be the same as keeping it at the default 14 days. If we do end up implementing this toggle, we may want to clearly explain this in a tooltip somewhere

Reply:
I think it's okay to have this ever present and have some value as a default – in some ways I think it may actually help teach our users that may be less familiar with funnels about what they're actually calculating when building this type of analysis.

A funnel without a conversion window is missing a key part of the question and I'm not sure if hiding / disabling it adds value given they can always set a very long conversion window, and it seems we'll always apply some sort of window in the back end.

Reply:
Sounds good! We can always iterate this in the future